### ISSUE TITLE
How can i set thumbnail of an image

### ISSUE THREAD

**Issue by santhoshs5** [rs/SDWebImage#2100]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

I have a local URL of an image which size is more than 1 MB how can i set thumbnail for it using 
sd_setImage(url,placeholder) method.

**Comment by bpoplauschi** [comment]
What do you mean by "set thumbnail"?
Also, it's very important that you fill in the issue template info above (I re-added the template for you).

**Comment by santhoshs5** [comment]
How can i get a thumbnail of an image from local URL.

**Comment by bpoplauschi** [comment]
For large images, you can use the `SDWebImageScaleDownLargeImages` option. This will make our library cache and return a scaled version of the image, if it's larger than a certain value that depends on the device.

**Comment by santhoshs5** [comment]
SDWebImageScaleDownLargeImages , is only for images or will it work with videos too?

**Comment by bpoplauschi** [comment]
SDWebImage does not handle videos, only images (including animated images like GIF or animated WebP).
